src/content/content.ts

```diff
@@ -0,0 +1,38 @@
+const headerComponent = {
+  serviceName: 'Notify',
+  links: {
+    logIn: 'Log in',
+    logOut: 'Log out',
+  },
+};
+
+const footerComponent = {
+  nhsEngland: 'NHS England',
+  supportLinks: 'Support links',
+  links: {
+    accessibilityStatement: 'Accessibility statement',
+    contactUs: 'Contact us',
+    cookies: 'Cookies',
+    privacyPolicy: 'Privacy policy',
+    termsAndCondition: 'Terms and conditions',
+  },
+};
+
+const mainLayout = {
+  title: 'NHS Notify - WEB IAM',
```

Comment:
We now have some guidance on this (https://nhsd-confluence.digital.nhs.uk/display/RIS/CCM-5686%3A+page+meta+information+and+urls+-+templating+UI )

It should be 'Sign in - Create and submit templates - NHS Notify'

Reply:
Done